scripts/build/electron-configure-package-win32.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+###
+# Copyright 2016 resin.io
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+set -u
+set -e
+
+function check_dep() {
+  if ! command -v $1 2>/dev/null 1>&2; then
+    echo "Dependency missing: $1" 1>&2
+    exit 1
+  fi
+}
+
+check_dep upx
+check_dep unzip
+
+function usage() {
+  echo "Usage: $0"
+  echo ""
+  echo "Options"
+  echo ""
+  echo "    -p <electron package>"
+  echo "    -n <application name>"
+  echo "    -d <application description>"
+  echo "    -v <application version>"
+  echo "    -r <application copyright>"
+  echo "    -l <application license file>"
+  echo "    -c <company name>"
+  echo "    -a <application asar (.asar)>"
+  echo "    -i <application icon (.ico)>"
+  echo "    -w <download directory>"
```

Comment:
I think I prefer to be explicit about it and ask it as an option rather than relying on the environment at all.